Automattic/jetpack
# Connection: hide the "Disconnect" modals on "Escape" press

If somebody feels uneasy about disconnecting, their first reflex might be to pres the "Escape" button to leave that uncomfortable modal without making any changes.
This PR adds a handler to close the "Disconnect" modals in Jetpack and "My Jetpack" on "Esc" press.

Please note that the PR is only handling the "Disconnect" modal itself, and doesn't handle any other modals.
This is made deliberately, as we need to approach it in a more orderly fashion.
The PR's goal is to address the immediate needs where the most impact can be made.

#### Changes proposed in this Pull Request:
* Hide the "Disconnect" modals on "Esc" button press

#### Other information:


#### Jetpack product discussion
N/a

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Connect Jetpack
2. Go to "Jetpack -> Dashboard" and click on "Manage site connection". Confirm that the modal gets closed upon an "Esc" button press.
3. Open the modal again and click "Disconnect". Keep pressing "Esc" white it's disconnecting, make sure the modal does NOT close.
4. You should see the "Successfully disconnected" modal afterwards. Make sure it also does NOT close on the "Esc" press.
5. Go to "My Jetpack", scroll down to "Connection" and click "Manage -> Disconnect Jetpack". You should see the "Are you sure you want to disconnect?" modal.
6. Press the "Esc" button, confirm that you go back to the "Manage your Jetpack connection" modal. Please note that the "Esc" button does NOT work for the "Manage your Jetpack connection" modal itself as per the deliberate decision.
7. Go back to the modal and click "Disconnect". Keep pressing "Esc" white it's disconnecting, make sure the modal does NOT close.
8. You should see the "Successfully disconnected" modal afterwards. Make sure it also does NOT close on the "Esc" press.
